From jeffm@suse.com  Mon Aug 10 16:23:48 2009
From: Jeff Mahoney <jeffm@suse.com>
Date: Tue, 04 Aug 2009 10:26:21 -0400
Subject: Staging: rtl8192su: fixup size comparison warning
To: Greg KH <gregkh@suse.com>
Message-ID: <20090804142710.150085378@suse.com>


sizeof(val) returns type unsigned long. count is already unsigned long, so
just use that. This fixes a compile warning.

Signed-off-by: Jeff Mahoney <jeffm@suse.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rtl8192su/ieee80211/ieee80211_module.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/rtl8192su/ieee80211/ieee80211_module.c
+++ b/drivers/staging/rtl8192su/ieee80211/ieee80211_module.c
@@ -264,7 +264,7 @@ static int store_debug_level(struct file
 			     unsigned long count, void *data)
 {
 	char buf[] = "0x00000000";
-	unsigned long len = min(sizeof(buf) - 1, (u32)count);
+	unsigned long len = min(sizeof(buf) - 1, count);
 	char *p = (char *)buf;
 	unsigned long val;
 
